examples/channel-alone/objectstore_channel_e2e.yaml

```diff
@@ -43,5 +43,5 @@ metadata:
   name: dev
   namespace: ch-obj
 data:
-  AccessKeyID: YWRtaW4= #<base64 encode username>
-  SecretAccessKey: cGFzc3dvcmRhZG1pbg== #<base64 encode password>
+  AccessKeyID: 8WzMANMhF1DNwfBxl52q #<base64 encode username>
+  SecretAccessKey: J8UQo0s5VY8cZmFw4u5DdC47ufB7XvMvDjT5Sg0q #<base64 encode password>
```

Comment:
is this the actual login info to the canary miniO instance? if so, we might want to mask it in a way...

Reply:
@ianzhang366  Thanks Ian, it's encoded now.

Reply:
@philipwu08  The example yaml is in the public channel repo.  Let's not use the real one and base64 encoded one. as it can be decoded easily. I think we can remain the previous fake one in the sample yaml